tests/analyser/analyser.cpp

```diff
@@ -974,4 +980,11 @@ TEST(Analyser, coverage)
     ast->setValue({});
     ast->setVariable(libcellml::Variable::create());
     ast->setParent(libcellml::AnalyserEquationAst::create());
+
+    auto parser = libcellml::Parser::create();
+    auto model = parser->parseModel(fileContents("analyser/coverage.cellml"));
+    printIssues(parser);
+
+    analyser->analyseModel(model);
+    printIssues(analyser);
```

Comment:
Is this for debugging purposes?

Reply:
Yes, it was. Clearly forgot to remove them! Done in 5612990f.